packages/multi-select-combo-box/theme/lumo/vaadin-multi-select-combo-box-chip-styles.js

```diff
@@ -13,27 +13,65 @@ import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themab
 
 const chip = css`
   :host {
-    display: inline-flex;
-    align-items: center;
-    align-self: center;
     font-family: var(--lumo-font-family);
```

Comment:
This can be inherited in my opinion.
```suggestion
    font-family: var(--lumo-font-family);
```

Reply:
Removed.